geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geohash/GeohashUtils.scala

```diff
@@ -359,6 +359,71 @@ object GeohashUtils
   }
 
   /**
+   * Find GeoHash instance with maximum precision that covers envelope defined by two points.
```

Comment:
`getMBGH` is no longer used anywhere, although I hesitate to remove it (there are plenty of "this could be useful later" type methods in this file). However the first implementation is a bit odd --- I wonder if it should be refactored to at least   return an Option[GeoHash], that way the null can be avoided as well as the require statement....of course the Option wrapping would need to be propagated to the other implementations.